dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1220,28 +1176,26 @@ def empty_like(
 
     """
 
-    if not isinstance(a, (dpnp.ndarray, dpt.usm_ndarray)):
-        pass
-    elif order not in ("C", "c", "F", "f", None):
-        pass
-    elif subok is not False:
-        pass
-    else:
-        _shape = a.shape if shape is None else shape
-        _dtype = a.dtype if dtype is None else dtype
-        _usm_type = a.usm_type if usm_type is None else usm_type
-        _sycl_queue = dpnp.get_normalized_queue_device(
-            a, sycl_queue=sycl_queue, device=device
-        )
-        return dpnp_container.empty(
-            _shape,
-            dtype=_dtype,
-            order=order,
-            usm_type=_usm_type,
-            sycl_queue=_sycl_queue,
+    if not dpnp.check_supported_arrays_type(a):
```

Comment:
won't `check_supported_arrays_type` raise an exception for unsupported type? why do we need to have a separate `raise(...)` below?